docs/UserGuide.md

```diff
@@ -851,18 +1071,26 @@ The `meet` command in EduMate is used to suggest suitable meet-up locations and
 
 </div>
 
+##### Scenario:
+
+After sorting your contacts, you decided to ask Tae Seong, Alex, Amirul and Kevin from your CCA to study for CS2100. 
+
+Problem is, you do not know when all of them are free! Luckily, you have `EduMate` to find out for you!
+
+You can execute the `study` variant of the `meet` command
+whilst referring to their index on the Person List and find out what are the most optimal places and the best possible time to meet and study for the final.
```

Comment:
Sentence is too long. Can be split by the first "and".

"You can execute the `study` variant of the `meet` command
whilst referring to their index on the Person List"

"From there, EduMate will recommend..."